bin/load_db_scxa_analytics.sh

```diff
@@ -3,9 +3,7 @@
 # This script takes the matrix market format files (normally available on
 # staging) for a single cell experiment and does the following steps:
 # - Transforms those files into a long (melted) table of experiment id,
-#   cell/run id, and expression. This takes care of avoiding large chunks of
```

Comment:
Any need to shorten this documentation here? Can you revert that deletion (lines 6 to 8) please unless there is a good reason.

Reply:
Sorry, I wanted to remove only the bit after `TODO`.